project/merge.go

```diff
@@ -28,18 +28,21 @@ var (
 	}
 )
 
-type rawService map[string]interface{}
-type rawServiceMap map[string]rawService
+// RawService is represent a Service in map form unparsed
+type RawService map[string]interface{}
```

Comment:
When moving to a `config` package, I think we should use a more *abstracted* name :stuck_out_tongue_closed_eyes: